dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/StoredDataSet.kt

```diff
@@ -0,0 +1,14 @@
+package org.dataland.datalandbackend.model
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+/**
+ * Class for defining the content of a data set     // TODO fix this doc
+ * @param name identifies the data set
+ * @param payload content of the stored data
+ */
+data class StoredDataSet(
+    @field:JsonProperty(required = true) val companyId: String,
+    @field:JsonProperty(required = true) val dataType: String,
+    @field:JsonProperty(required = true) val data: String
+)
```

Comment:
Grundsätzlicher Kommentar:
Sehr viele JavaDocs des Backends sind in diesem Branch aktuell fehlerhaft/unsinnig.
Im Branch DALA-65 habe ich das jedoch gefixt. Hier nicht.

Reply:
Hier gefixt.   Bei Merge in DALA-65 werde ich die Konflikte auflösen müssen